lib/character.js

```diff
@@ -34,24 +24,12 @@ class Character extends Lib {
 		data
 	}
 	*/
-	get(id, params = {}) {
-		return new Promise((resolve, reject) => {
-			if(!id)
-				reject(this.throwError('character.get()', 'an ID'))
-
-			params.extended = params.extended ? 1 : 0
```

Comment:
This check should either remain or (preferably) be moved inside the req function definition